core/orchestrator_core.go

```diff
@@ -41,6 +41,17 @@ const (
 // recordTiming is used to record in Prometheus the total time taken for an operation as follows:
 //   defer recordTiming("backend_add")()
 // see also: https://play.golang.org/p/6xRXlhFdqBd
+// It returns an error if the backend update is invalid.
```

Comment:
Currently the whole doc for that function would be detected as "fair play" and possibly rewritten when the hash changes.

Reply:
Does that mean a regeneration would overwrite the original manually-generated content?  That would seem problematic.  I wondered if your ```It returns``` language was a cue that anything before that was to be preserved.

Reply:
It might overwrite the original manual content yes. But possibly that content became obsolete with the code change yes? The human writing the new code would decide whether the generated doc is acceptable. If it isn't they would updated it manually and remove the autogenerated tag?

Reply:
The hash line starts with ```// --```.  If the script placed another delimiter line between preexisting & generated content, it could preserve the former during an update.  Thoughts, @ntap-rippy?